teleport/metadata.csv

```diff
@@ -1,93 +1,130 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
-teleport.audit.failed_disk_monitoring,count,,,,Number of times disk monitoring failed.,0,teleport,,
-teleport.audit.failed_emit_events,count,,,,Number of times emitting audit event failed.,0,teleport,,
-teleport.audit.percentage_disk_space_used,gauge,,,,Percentage disk space used.,0,teleport,,
-teleport.audit.server_open_files,gauge,,,,Number of open audit files,0,teleport,,
+teleport.common.certificate_mismatch,count,,,,Number of times there was a certificate mismatch,0,teleport,,
+teleport.auth.audit_log.failed_disk_monitoring,count,,,,Number of times disk monitoring failed.,0,teleport,,
```

Comment:
Maybe you should standardize adding a period at the end of the description or not adding it? (personally I'm in favor of not adding it, but I don't know if there is a guideline)

Reply:
Nice catch, I'll remove them!